shuup/front/templates/shuup/front/checkout/_vertical_phases.jinja

```diff
@@ -89,7 +89,7 @@
             }
         };
         $(function() {
-            $("form").on("submit", function(e) {
+            $("form").on("submit","[id='addresses'], [id='methods'], [id='payment-form'], [id='confirm_form']", function(e) {
```

Comment:
Can we not expect here some id:s for the forms.

Why don't you for example add some class here like `vertical-checkout-phase` or similar then update that jquery selector to something like `$('.vertical-checkout-phase').find("form")` that way we trigger this on submit for forms inside that ajax content.